client-src/elements/chromedash-report-external-reviews-page.js

```diff
@@ -0,0 +1,247 @@
+// @ts-check
+import {Task} from '@lit/task';
+import '@shoelace-style/shoelace';
+import {LitElement, css, html, nothing} from 'lit';
+import {SHARED_STYLES} from '../css/shared-css.js';
+
+/**
+ * @typedef {import('chromestatus-openapi').ExternalReviewsResponse} ExternalReviewsResponse
+ * @typedef {import('chromestatus-openapi').OutstandingReview} OutstandingReview
+ * @typedef {import('chromestatus-openapi').OutstandingReviewCurrentStageEnum} Stage
+ */
+
+/** Array.sort() comparison helper function ordering numbers ascending and putting undefined last.
+ *
+ * @param {number | undefined} a
+ * @param {number | undefined} b
+ * @returns {number}
+ */
+function ascendingNumberUndefinedLast(a, b) {
+  if (a === b) return 0;
+  if (a === undefined) return 1;
+  if (b === undefined) return -1;
+  return a - b;
+}
+
+/** Array.sort() comparison function to order outstanding reviews by descending urgency.
+ *
+ * Reviews' features must be within the same stage.
+ *
+ * @param {OutstandingReview} a
+ * @param {OutstandingReview} b
+ * @returns {number}
+ */
+function compareOutstandingReview(a, b) {
+  console.assert(a.current_stage === b.current_stage);
```

Comment:
Is this console.assert needed?

Reply:
I think I'd like to either keep it or turn it into an exception, although I'll add a better error message. The function doesn't do a reasonable sort on features whose current stages are different, so if the code ever changes to ask it to, we should notice that somehow. I lean toward a console message rather than an exception, because a bad sort doesn't make the page useless, and an exception probably would.